app/mailers/user_mailer.rb

```diff
@@ -11,8 +11,17 @@ def work_order_notification(user, job, work_order)
     @user = user
     @job = job
     @work_order = work_order
+    subject = "#{@user.full_name}, you have been sent a work order for job #{@job.id}, #{@job.name}. Franchise: #{@job.franchise.try(:name)}."
 
-    mail(to: @user.email, subject: "#{@user.full_name}, you have been sent a work order for job #{@job.id}, #{@job.name}. Franchise: #{@job.franchise.try(:name)}.")
+    @mail = mail(to: @user.email, subject: subject)
+
+    MailLog.create(
+      name: @user.full_name,
+      email: @user.email,
+      subject: subject,
+      body: @mail.body.encoded,
+      mail_loggable: work_order
```

Comment:
Yes, it's just a polymorphic relationship: https://github.com/Service-Alliance/Service_Phase_1/pull/15/files#diff-53f2bd956aa320716585cca9bf14be90L411 :)